apps/avifenc.c

```diff
@@ -1813,6 +1858,17 @@ int main(int argc, char * argv[])
 #if defined(AVIF_ENABLE_EXPERIMENTAL_JPEG_GAIN_MAP_CONVERSION)
         } else if (!strcmp(arg, "--ignore-gain-map")) {
             settings.ignoreGainMap = AVIF_TRUE;
+        } else if (!strcmp(arg, "--qgainmap")) {
+            NEXTARG();
+            int qualityGainMap = atoi(arg);
+            if (qualityGainMap < AVIF_QUALITY_WORST) {
```

Comment:
Please use AVIF_CLAMP

Reply:
It's defined in internal.h which is not available in apps/

Reply:
Ah, I thought it was internal to the project, not the lib...